From 563f31ad081038ece8b5a5b525dd516fcd6ad0fd Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Sat, 13 Mar 2010 19:52:56 +1000
Subject: [PATCH] qxl: remove asserts that make no sense anymore

Not sure why nobody noticed these, once I dropped qxl into Fedora it asserted.
Looks like they were missed when p->Private was changed to an index.

Signed-off-by: Dave Airlie <airlied@redhat.com>
---
 src/qxl_driver.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/src/qxl_driver.c b/src/qxl_driver.c
index 25fe59b..187e581 100644
--- a/src/qxl_driver.c
+++ b/src/qxl_driver.c
@@ -1231,9 +1231,6 @@ qxl_valid_mode(int scrn, DisplayModePtr p, Bool flag, int pass)
 
     p->Private = (void *)(unsigned long)mode_idx;
 
-    assert (((struct qxl_mode *)p->Private)->x_res == p->HDisplay);
-    assert (((struct qxl_mode *)p->Private)->y_res == p->VDisplay);
-    
     return MODE_OK;
 }
 
-- 
1.6.6

